components/admin_console/data_retention_settings/channel_list/channel_list.tsx

```diff
@@ -200,9 +200,9 @@ export default class ChannelList extends React.PureComponent<Props, State> {
                 );
             }
             return {
-                cells: {
-                    id: channel.id,
-                    name: (
+                cells: new Map<string, React.ReactNode>([
```

Comment:
Creating maps seems a bit more troublesome, but doesn't seem a big deal. But I have to admit that migrating these is tedious.